.tekton/kn-client-cli-artifacts-115-pull-request.yaml

```diff
@@ -26,6 +26,7 @@ spec:
         - GO_RUNTIME=registry.access.redhat.com/ubi8/ubi-minimal
         - JAVA_BUILDER=registry.access.redhat.com/ubi8/openjdk-21
         - JAVA_RUNTIME=registry.access.redhat.com/ubi8/openjdk-21-runtime
+        - CLI_ARTIFACTS=brew.registry.redhat.io/rh-osbs/openshift-serverless-1-kn-cli-artifacts-rhel8:1.15.0
```

Comment:
This will be overridden by konfluxgen. I guess you need to add this here: https://github.com/openshift-knative/hack/blob/main/config/client.yaml as an image override (like here: https://github.com/openshift-knative/hack/blob/4183948d2539a0e8b1cd67d443ace45e65ea248f/config/serverless-operator.yaml#L8-L16)